src/check.js

```diff
@@ -71,6 +73,19 @@ function getDependencies(dir, filename, deps, parser, detectors) {
         .flatten()
         .uniq()
         .map(requirePackageName)
+        .thru(_dependencies => (
```

Comment:
I'm sure it must be possible to factor this more cleanly, though it's not clear to me what the best way of extracting an API to allow this kind of processing would be, or if that would really be worth the effort.